### PR TITLE
Abb 156

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New records Abb 156 (quite a difficult manuscript, still not all problems have been solved), minor changes in others (ignore MY 018).

### CODE REVIEW

File: ParisBNF/abb/BNFabb156.xml [hunk 1]
```diff
@@ -0,0 +1,627 @@
...
+                     <layoutDesc>
+                       <layout columns="2" writtenLines="28 30">
+                            <locus from="3ra" to="130vb"/>
+                          
+                           <ab type="pricking">Pricking is partly visible.</ab>
+                          <ab type="ruling">Ruling is partly visible.</ab>
+                          <ab type="pricking">Pricking is not visible.</ab>
+                          <ab type="ruling">Ruling is not visible.</ab>
+                          <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
+                          <note>The leaves <locus target="#1 #2"/> appear to have been neither pricks nor ruling.</note>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb156.xml:361]: "to contain neither"  or "appear not to have been pricked..."?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb156.xml [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+                  
+                  <msItem xml:id="ms_i4">
+                     <locus from="108va" to="130vb"/>
+                     <title type="complete" ref="LIT3183Commen">Commentaries on the Books of the Minor Prophets, in Gǝʿǝz</title>
+                  <textLang mainLang="gez"/>                     
+                  
+             
+                  <msItem xml:id="ms_i4.1">
+                     <locus from="108va" to="111vb"/>
+                     <title type="complete" ref="LIT3183Commen#Hosea">Interpretation of the kings' names mentioned in <ref type="work" corresp="LIT3144Hosea">Book of Hosea</ref></title>
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply ParisBNF/abb/BNFabb156.xml:248]: We cannot have mark up inside of `title`, right, cross-references should be moved to `note`

File: ParisBNF/abb/BNFabb156.xml [hunk 3]
```diff
@@ -0,0 +1,627 @@
...
+                  <collection>d'Abbadie</collection>
+                 <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b100873516">BnF Éthiopien d'Abbadie 156</idno>
+                    </msIdentifier>
+               <msContents>
+                  <summary>
+                  </summary>
+                  
+                  <msItem xml:id="ms_i1">
+                     <locus from="3ra" to="19ra"/>                     
+                     <title type="complete" ref="LIT6824CommentariesOldTest"><foreign xml:lang="gez">Tǝrgʷāme</foreign>-commentaries on the books of the Octateuch</title>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb156.xml:39]: please no mark-up inside of `title`

File: ParisBNF/abb/BNFabb156.xml [hunk 4]
```diff
@@ -0,0 +1,627 @@
...
+                        <incipit xml:lang="gez"><hi rend="rubric">ስብሐት፡ ለእግዚአብሔር፡ መብርሄ፡ ልቡና፡ ንጽሕፍ፡</hi> ትርጓሜ፡ ኦሪት፨ 
+                        ኦሪት፡ ብሂል፡ ፍትሕ፡ ወሥርዓት፡ ወሕግ<hi rend="rubric">ሂ፡ ይትበሀል። ወቦ፡ እለ፡ እቤሉ፡ አእምሮ፡ ዘይፈልጥ፡</hi>ሠናየ፡ እምእኩይ። 
+                        ወቦ፡ እለ፡ ይብሉ፡ መጽሔት፡ ዘይሬእዩ፡ ባቲ፡ ግብረ፡ እግዚአብሔር። እስመ፡ በሌሊት፡ እሁድ፡ ፈጠረ፡ እግዚአብሔር፡ ፯ፍጥረታተ፡ በአርምሞ። 
+                        ተፈጽመ፡ ፯ሥርዋት፡ ዘውእቶሙ፡ መላእክት፡ ሰማይ፡ 
+                        </incipit>
+                     </msItem>
+
+                              <msItem xml:id="ms_i1.2">
+                                <locus from="4ra" to="9ra"/>
+                                 <title type="complete" ref="LIT4117Interpretation">Interpretation and symbolic commentary on <ref type="work" corresp="LIT1546Genesi">Genesis</ref></title>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb156.xml:54]: no mark-up inside of `title` here and elsewhere

File: ParisBNF/abb/BNFabb156.xml [hunk 5]
```diff
@@ -0,0 +1,627 @@
...
+
+                                 <msItem xml:id="ms_i1.3">
+                                    <locus from="9ra" to="12vb"/>
+                                    <title type="complete"  ref="LIT6824CommentariesOldTest#Exodus">Commentaries on <ref type="work" corresp="LIT1367Exodus">Exodus</ref>
+                                    </title>
+                                    <incipit xml:lang="gez"><hi rend="rubric">በካልእ፡ ዓመት፡ እምአመ፡ ወፅኡ፡ እስራኤል፡ ተጽ</hi>ሕፈ፡ ኦሪት፡ ዘፀአት፡ 
+                                       ዝውእቱ፡ ትርጓሜ፡ ቃሉ፡ ኅቡእ፨ በበአዕፃደቲሆሙ፨ ወዓዲ፡ በበደባትሪሆሙ፨ ወዓዲ፡ በበነገዶሙ፨ ሕቁራነ፡ ዘ፡ ፍድፉዳነ፤ ወቦ፡ እ፡ ይ፡ ምኑናነ። ሳዙዝ፡ ዘ፡ ጽኑዕ፤ 
+                                       ወቦ፡ እ፡ ይ፡ ብዙሳ፨ ወመዓልት፡ ምድር፡ እምኖሙ፡ ዘ፡ ግዝረቶሙ፨</incipit> <explicit xml:lang="gez">ደመና፡ መዓልተ፡ ወእሳት፡ ሌሊተ፨ ተፈጸመ፡ በዝየ፡ ትርጓሜ፡ ኦሪት፡ ዘፀአት፡ 
+                                       በሰላመ፡ እግዚአብሔር፡ ጸባኦት፡ አሜን፨</explicit>
+                                    <note>Small excerpts from Exodus are supplied with explanations separated through ዘ,
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb156.xml:72]: here you can use the reference to Exodus, not in title

File: ParisBNF/abb/BNFabb156.xml [hunk 6]
```diff
@@ -0,0 +1,627 @@
...
+                                    መምህር፨ ዘአስተጋብእዎ፡ ማእምራነ፡ መጻሕፍት፨ ወረባሁሰ፡ ለዝመጽሐፍ፡ ይሜህር፡ ወይተረጕሞ፡ ቃላተ፡ ወነገራተ፡ 
+                                    ዘኦሪት፡ እንተ፡ የዓፅብ፡ ወይስአን፨ ዘይትሀባእ፡ ወዘይሤውር፡ ለዘ፡ ኢየአምር፡ ብእሲ፨ ወትርጓሜሁሰ፡ በኀበ፡ ይነግር፡ 
+                                    ፩ቃለ፡ በ፫ጾታ፡ ወበበ፪፡ ስብሐት፡ ለእግዚአብሔር፡ በአፈ፡ ኵሉ፡ ፍጥረት፡ <gap reason="ellipsis" resp="DN"/> <hi rend="rubric">ኦሪት፡ ዘልደት፡</hi>
+                                    ኦሪት፡ ብሂል፡ ሕግ፡ ዘልደት፡ ብሂል፡ ዘፍጠት፨ 
+                                 </incipit>
+                              </msItem>
+
+                                 <msItem xml:id="ms_i1.3">
+                                    <locus from="9ra" to="12vb"/>
+                                    <title type="complete"  ref="LIT6824CommentariesOldTest#Exodus">Commentaries on <ref type="work" corresp="LIT1367Exodus">Exodus</ref>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb156.xml:66]: the `title `is converted to a link, and there can be no link inside of a link, so please no mark up inside of title

File: ParisBNF/abb/BNFabb156.xml [hunk 7]
```diff
@@ -0,0 +1,627 @@
...
+             
+                  <msItem xml:id="ms_i4.1">
+                     <locus from="108va" to="111vb"/>
+                     <title type="complete" ref="LIT3183Commen#Hosea">Interpretation of the kings' names mentioned in <ref type="work" corresp="LIT3144Hosea">Book of Hosea</ref></title>
+                     <textLang mainLang="gez"/>                     
+                  </msItem>
+                     
+                     <msItem xml:id="ms_i4.2">
+                        <locus from="111vb" to="113vb"/>
+                        <title type="complete" ref="LIT3183Commen#Amos">Commentary on <ref type="work" corresp="LIT3145Amos">Book of Amos</ref></title>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb156.xml:254]: mark up

File: ParisBNF/abb/BNFabb156.xml [hunk 8]
```diff
@@ -0,0 +1,627 @@
...
+                        <textLang mainLang="gez"/>                     
+                     </msItem>
+                  
+                  <msItem xml:id="ms_i4.3">
+                     <locus from="113vb" to="115ra"/>
+                     <title type="complete" ref="LIT3183Commen#Micah">Commentary on <ref type="work" corresp="LIT3146Micah">Book of Micah</ref></title>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb156.xml:260]: +

File: ParisBNF/abb/BNFabb156.xml [hunk 9]
```diff
@@ -0,0 +1,627 @@
...
+                     <textLang mainLang="gez"/>
+                     <incipit xml:lang="gez"><hi rend="rubric">ቃለ፡᎓እግዚአብሔር፡ ዘኮነ፡ ኀበ፡ ሚክያስ፡ ዘ፡ ኮነ፡ </hi>መንፈሰ፡ ትንቢት፡ በአፈ፡ ሚክያስ። ዘርእየ፡ በእንተ፡ ሰማርያ፡ ወኢየሩሳሌም፡ ዘ፡ ተነበየ፡ ለቤተ፡ ይሁዳ፨</incipit>
+                  </msItem>
+                  
+                  <msItem xml:id="ms_i4.4">
+                     <locus from="115ra" to="116ra"/>
+                     <title type="complete" ref="LIT3183Commen#Joel">Commentary on <ref type="work" corresp="LIT1689Joel">Book of Joel</ref></title>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb156.xml:267]: +

File: ParisBNF/abb/BNFabb156.xml [hunk 10]
```diff
@@ -0,0 +1,627 @@
...
+                     <textLang mainLang="gez"/>                     
+                  </msItem>
+                  
+                  <msItem xml:id="ms_i4.5">
+                     <locus from="116ra" to="116va"/>
+                     <title type="complete" ref="LIT3183Commen#Obadiah">Commentary on <ref type="work" corresp="LIT3147Obadiah">Book of Obadiah</ref></title>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb156.xml:273]: here as above, also below

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb156.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb156.xml:248]: Thank you! One indication like that above is enough, no need to repeat it many times (as it probably costs time).

**Review by thea-m — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**